apps/client/src/layout/components/footer/footer.css

```diff
@@ -0,0 +1,34 @@
+/**Custom footer component CSS*/
+
+.dashboard-footer-container { 
+    z-index: 1002; 
+    position:sticky;
+    bottom:0;
+    display: flex;
+    align-items: center;
+    justify-content: space-between;
+    padding: 8px 30px;
+    font-size: 14px;
+}
+
+.dashboard-footer-container ul {
+    display: flex;
+    flex-wrap: wrap;
+    list-style: none;
+    margin: 0;
+    padding: 0;
+    gap: 24px;
+}
+
+.dashboard-footer-container ul li a {
+    color: #ffff !important;
+}
+
+.dashboard-footer-container ul li a:hover {
+    color: #9ba7b6 !important;  
```

Comment:
same here

Reply:
to style hover with inline CSS in React we need to add additional wrapper component and conditionally set inline styles using state on onMouseEnter and onMouseLeave events. Do we really need that ? @mnischay